helm/charts/templates/ingress.yaml

```diff
@@ -1,4 +1,5 @@
-{{- if and (.Values.jxRequirements.ingress.domain) (not .Values.knativeDeploy) }}
+{{- $host := (.Values.ingress.host | default (printf "%s%s%s" .Values.service.name .Values.jxRequirements.ingress.namespaceSubDomain .Values.jxRequirements.ingress.domain)) | quote }}
```

Comment:
$host should be set within the if statement, as the other variables only needed if the ingress is actually created.